### DISCUSSION TITLE
Scaling SVG before rasterizing

### DISCUSSION THREAD

**Discussion by Edward-Russel** [pixijs/pixijs#10458] [v8 General]
Hi! It used to be possible to scalify SVGs before rasterisation **in v7** like this:
```js
const svg = new SVGResource("example.svg", { scale: 2 });
const texture = Texture.from(svg);
```
Is it possible to do this kind of scaling in v8 now? If I dynamically scale the SVG before rasterising it, it will look sharm at the size I need.

**Comment by edmeow** [comment]
so here is answer to my question. thanks https://pixijs.com/8.x/guides/basics/svg
